### PR TITLE
Release v1.2.0

### PR DESCRIPTION
**Pull request by jhermsmeier**
**Checklist:**

- [x] Bump the version number in the `package.json`'s `version` property
- [x] Bump the version number in the `npm-shrinkwrap.json`'s `version` property
- [x] Add a new entry to `CHANGELOG.md` by running `make changelog`
- [x] Re-take `screenshot.png` so it displays the latest version in the bottom
right corner
- [x] Revise the `updates.semverRange` version in `package.json`
- [x] Commit the changes with the version number as the commit title, including the
`v` prefix, to `master`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,35 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## v1.2.0 - 2017-11-15
+
+### Features
+
+- Add the progress and status to the window title.
+- Add a sudo-prompt upon launch on Linux-based systems.
```
Comment (lurch) [inline CHANGELOG.md]: This 'feature' got reverted

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,35 @@
...
+
+- Remove Linux elevation meant for usbboot.
+- Fix `LIBUSB_ERROR_NO_DEVICE` error at the end of usbboot.
+- Gracefully handle scenarios where a USB drive is disconnected halfway through the usbboot procedure.
+- Make sure the progress button is always rounded.
+
+### Misc
+
+- Add the Python version (2.7) to the CONTRIBUTING doc.
+- Remove duplicate debug enabling in usbboot module.
```
Comment (lurch) [inline CHANGELOG.md]: Not sure this really warrants an error in the Changelog?

**Comment by lurch** [comment]
Should " Revise the updates.semverRange version in package.json" in your comment above have a " (if necessary)" on the end? ;-)

**Comment by lurch** [comment]
Given the ongoing discussions about electron-builder in Gitter, I guess we might end up publishing v1.2.1 rather than v1.2.0 ?

**Comment by jhermsmeier** [comment]
> According to Github, the screenshot is now half the size that it was?

Yeah it is, because my screen only has a device-pixel-ratio of 1.5 or so – I guess @jviotti could send me a 2x screenshot to put in here

**Comment by jviotti** [comment]
2x screenshot:

screen shot 2017-11-17 at 10 53 06 am

**Pull request merged**